TF_Settings_Web/src/TauriUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { open } from '@tauri-apps/api/shell';
+import { appWindow } from '@tauri-apps/api/window';
+
+export const isDesktop = () => '__TAURI_METADATA__' in window;
```

Comment:
Is this documented in Tauri docs? Could do with a comment explaining why/how this works.
👀

Reply:
It's documented on their changelog 
`Breaking change: Move __currentWindow and __windows values from window.__TAURI__ to window.__TAURI_METADATA__`
but I found that solution via the forums and then console logging window while in a Tauri environment.

Will add a comment